source/Octopus.Client/Octopus.Client.csproj

```diff
@@ -5,32 +5,26 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <AssemblyName>Octopus.Client</AssemblyName>
     <GenerateAssemblyTitleAttribute>false</GenerateAssemblyTitleAttribute>
-    <PackageId>Octopus.Client</PackageId>
```

Comment:
All the package stuff was duplicated in the .nuspec file, and this was generating warnings.